vitest.setup.ts

```diff
@@ -5,9 +5,10 @@ import { cleanup } from '@testing-library/react'
 import { setupServer } from 'msw/node'
 import { afterEach } from 'vitest'
 
+import { productHandlers } from '@/lib/msw/product.ts'
 import { vendorHandlers } from '@/lib/msw/vendor.ts'
 
-const server = setupServer(...vendorHandlers)
+const server = setupServer(...vendorHandlers, ...productHandlers)
```

Comment:
If the handler grows, it might become too verbose

Reply:
We might implement something like this

```
import { productHandlers } from './product'
import { vendorHandlers } from './vendor'

export const handlers = [
  ...vendorHandlers,
  ...productHandlers,
]
```

Reply:
Yap good idea, we can do that I think 👍